lib/middleware/authorization.js

```diff
@@ -109,32 +111,32 @@ module.exports = function(server, options, validate, immediate) {
   }
   options = options || {};
   immediate = immediate || function (client, user, done) { return done(null, false); };
-  
-  if (!server) { throw new TypeError('oauth2orize.authorization middleware requires a server argument'); }
-  if (!validate) { throw new TypeError('oauth2orize.authorization middleware requires a validate function'); }
-  
+
+  if (!server) { throw new TypeError(g.f('{{oauth2orize.authorization}} middleware requires a server argument')); }
+  if (!validate) { throw new TypeError(g.f('{{oauth2orize.authorization}} middleware requires a validate function')); }
```

Comment:
I believe even `middleware ` should not be translated; I leave the decision up to you.